client/src/components/App.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReviewList from './ReviewList.jsx';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+  this.state = {
+    listOfReviews: [],
+    reviewers: []
+  };  
+  }
```

Comment:
nit picky: can you fix this indent? :P 
linters have been drilled into my brain.

Reply:
will do! thanks for the catch. i've installed linters, but not sure if they're working?